wikitemplate-ReducedDesktop.md

```diff
@@ -13,9 +13,9 @@
 
 - [ ] Verify that none of the reward endpoints are being contacted when a user visits a media publisher (`youtube.com`, `reddit.com`, `twitter.com`, `github.com`) and hasn't interacted with rewards
   - [ ] Verify that `rewards.brave.com`, `pcdn.brave.com`, `grant.rewards.brave.com` or `api.rewards.brave.com` are not being contacted
-- [ ]  Verify account balance shows correct BAT and USD value
-- [ ]  Verify you are able to restore a wallet
-- [ ]  Verify actions taken (claiming grant, tipping, auto-contribute) display in wallet panel
+- [ ]  Verify BAT balance shows correct BAT and USD value
```

Comment:
just curious if this should say "Rewards balance" rather than "BAT balance"

Reply:
@srirambv can adjust this wording 👍🏻 

will create a follow up.